demo/kubernetes/rook-client/rook-client.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rook-client
+spec:
+  containers:
+  - name: rook-client
+    image: quay.io/rook/k8s-demo
+    imagePullPolicy: IfNotPresent
+    command: ["sleep", "36500d"]
```

Comment:
No, if we did use it we would probably need to include the code and build it ourselves.